client/components/mypaper.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable comma-dangle */
```

Comment:
You can turn this off in `.eslintrc` if you really want to 😂

Reply:
For some reason, the linter was only freaking out about the commas in that one module. If I had dangling commas, the linter would say it shouldn't be there, but when I got rid of it, the linter would tell me something else and would put in that comma automatically when I saved.
![Screen Shot 2021-07-22 at 3 29 38 PM](https://user-images.githubusercontent.com/82540886/126717380-5b7ae4d3-aec0-4b9d-9722-bd6ab8308bef.png)
![Screen Shot 2021-07-22 at 3 29 45 PM](https://user-images.githubusercontent.com/82540886/126717381-6eb7bfbd-e8e0-4b6c-a58b-a9bd809ec725.png)